experimental/examples/bubblebench/editable-shared-tree/tests/demo.test.ts

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// import { globals } from "../jest.config";
+
+// The current state of editable-tree bubble bench will not run successfully.
+describe("Bubblebench", () => {
+    describe("SharedTree", () => {
+        // beforeAll(async () => {
+        //     // Wait for the page to load first before running any tests
+        //     // so this time isn't attributed to the first test
+        //     await page.goto(globals.PATH, { waitUntil: "load", timeout: 0 });
+        // }, 45000);
+
+        // beforeEach(async () => {
+        //     await page.goto(globals.PATH, { waitUntil: "load" });
+        //     await page.waitFor(() => window["fluidStarted"]);
+        // });
```

Comment:
We should not commit commented out code unless that code is part of useful documentation (and even then doing so should be limited because commented out code is hard to maintain across refactors and such). This should probably be removed or enabled.

Reply:
Deleted in latest commit aeb65

Reply:
it looks like there are no longer any tests. Did you intend to delete the tests entirely? I think it would be good to have a test to help make sure this app stays working.

Reply:
That commented out test I deleted was the only test available and it fails in the current bubble bench state. So, yes I removed it for now.

Reply:
If we need it back we can simply refer to other bubble bench examples to add it back it is a carbon copy of the other bubble bench implementation test files.

Reply:
if you want to keep but disable the tests, using ".skip" is preferable to commenting them out. More documentation about why they are failing would be nice too.

Reply:
Okay. I added the tests back and added an explanatory comment in recent commit 8a204